state/modelsummaries_test.go

```diff
@@ -0,0 +1,390 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state_test
+
+import (
+	"sort"
+	"time"
+
+	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/names.v2"
+	"gopkg.in/mgo.v2/bson"
+	"github.com/juju/utils"
+	//"gopkg.in/macaroon.v1"
```

Comment:
If we don't need these imports, let's delete them altogether rather than comment out :)

Reply:
indeed.